astropy/units/quantity.py

```diff
@@ -7,32 +7,30 @@
 """
 
 
-# Standard library
-import re
 import numbers
-from fractions import Fraction
 import operator
+# Standard library
+import re
 import warnings
+from fractions import Fraction
 
 import numpy as np
 
-# AstroPy
-from .core import (Unit, dimensionless_unscaled, get_current_unit_registry,
-                   UnitBase, UnitsError, UnitConversionError, UnitTypeError)
-from .structured import StructuredUnit
-from .utils import is_effectively_unity
-from .format.latex import Latex
+from astropy import config as _config
 from astropy.utils.compat.misc import override__dir__
+from astropy.utils.data_info import ParentDtypeInfo
 from astropy.utils.exceptions import AstropyDeprecationWarning, AstropyWarning
 from astropy.utils.misc import isiterable
-from astropy.utils.data_info import ParentDtypeInfo
-from astropy import config as _config
-from .quantity_helper import (converters_and_unit, can_have_arbitrary_unit,
-                              check_output)
-from .quantity_helper.function_helpers import (
-    SUBCLASS_SAFE_FUNCTIONS, FUNCTION_HELPERS, DISPATCHED_FUNCTIONS,
-    UNSUPPORTED_FUNCTIONS)
 
+# AstroPy
```

Comment:
This comment is confusing - just imported lots of stuff from astropy.  Maybe `# local (astropy.units)`

Reply:
Done.